### PR TITLE
Fix for hab 0.33.2 - do not provide '/v1/depot' to Hab URL

### PR DESCRIPTION
**Pull request by tduffield**
Signed-off-by: Tom Duffield <tom@chef.io>

### Description

Do not provide 'v1/depot' to URL - addresses https://github.com/habitat-sh/habitat/pull/3239. 

### Check List

- [ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

**Comment by jtimberman** [comment]
I think we should pin the version of Hab that we're installing by default in the `hab_install` resource. This will prevent the cookbook from installing a later version than it has been updated to support, in case these kinds of changes are made in the future before a 1.0 release.

File: test/fixtures/cookbooks/test/recipes/package.rb [hunk 1]
```diff
@@ -13,5 +13,9 @@
 end
 
 hab_package 'core/hab-sup' do
-  depot_url 'http://app.acceptance.habitat.sh/v1/depot'
+<<<<<<< HEAD
```
Comment (tas50) [inline test/fixtures/cookbooks/test/recipes/package.rb]: Oops

**Comment by tas50** [comment]
@tduffield You have a merge artifact there

**Comment by tduffield** [comment]
@tas50 yeah - I blame @jtimberman. He is working on adding another commit.

**Comment by jtimberman** [comment]
Blame taken! Git still baffles me after 10 years ;).

@tduffield this is ready for review per our discussion

**Comment by tduffield** [comment]
The travis failures are caused by an issue with that repo that the appropriate team is aware of.

**Comment by tduffield** [comment]
The package travis failures are caused by an issue with that repo that the appropriate team is aware of. @jtimberman we need to update the supervisor resource based on the failures I'm seeing in Travis

File: resources/install.rb [hunk 2]
```diff
@@ -20,16 +20,21 @@
...
 
 property :install_url, String, default: 'https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh'
-property :depot_url, String
+property :bldr_url, String
 property :version, String
 property :channel, String
 
 action :install do
+  if new_resource.version
+    Chef::Log.warn("Do not specify a version of Habitat with the 'hab_install' resource.")
```
Comment (tduffield) [inline resources/install.rb]: Should we mention here that the version is being ignored?

**Pull request merged**